src/Tizen.Telephony/Tizen.Telephony/Call.cs

```diff
@@ -97,7 +96,6 @@ public CallPreferredVoiceSubscription PreferredVoiceSubscription
         public IEnumerable<CallHandle> GetCallHandleList()
         {
             uint count;
-            _callList = new IntPtr();
             _list.Clear();
             TelephonyError error = Interop.Call.GetCallList(_handle, out count, out _callList);
```

Comment:
According to Native API document, you should release _callList after used, and `SafeCallList` seem to do it. Why do you remove it?
```
/**
 * @brief Gets the list of the current call.
 *
 * @since_tizen @if MOBILE 2.4 @elseif WEARABLE 3.0 @endif
 * @privlevel public
 * @privilege %http://tizen.org/privilege/telephony
 *
 * @remarks You must release @c call_list using telephony_call_release_call_list().
 *
 * @param[in] handle The handle from telephony_init()
 * @param[out] count Count of the existing calls
 * @param[out] call_list List of call information for existing calls
 *
 * @return @c 0 on success,
 *         otherwise a negative error value
 *
 * @retval #TELEPHONY_ERROR_NONE              Successful
 * @retval #TELEPHONY_ERROR_INVALID_PARAMETER Invalid parameter
 * @retval #TELEPHONY_ERROR_PERMISSION_DENIED Permission denied
 * @retval #TELEPHONY_ERROR_NOT_SUPPORTED     Not supported
 * @retval #TELEPHONY_ERROR_OPERATION_FAILED  Operation failed
 */
int telephony_call_get_call_list(telephony_h handle, unsigned int *count, telephony_call_h **call_list);
```

Reply:
I will add ReleaseCallList method as invisible on next patchset.